src/quo/components/buttons/logout_button/component_spec.cljs

```diff
@@ -0,0 +1,35 @@
+(ns quo.components.buttons.logout-button.component-spec
+  (:require
+    [quo.components.buttons.logout-button.view :as logout-button]
+    [test-helpers.component :as h]))
+
+(h/describe "button tests"
+  (h/test "default render of logout button component"
+    (h/render [logout-button/view
+               {:accessibility-label "test-button"}])
+    (h/is-truthy (h/get-by-label-text "test-button")))
```

Comment:
We should avoid using a string in the `accessibility-label`.  Use `keyword` as they are cached.

https://github.com/status-im/status-mobile/blob/develop/doc/new-guidelines.md#accessibility-labels